Frameworks/UIKit/UIImage.mm

```diff
@@ -123,6 +122,11 @@ + (UIImageCachedObject*)cacheImage:(UIImage*)image withName:(NSString*)name {
     return [obj autorelease];
 }
 
+static inline CGImageRef getImage(UIImage* self) {
+    RETURN_NULL_IF(!self);
```

Comment:
What are the circumstances?  If the creation indeed failed, we shouldn't typically get here.  

---
In reply to: [84755738](https://github.com/Microsoft/WinObjC/pull/1185#discussion_r84755738) [](ancestors = 84755738,84748990)